lib/builderator/control/version.rb

```diff
@@ -32,6 +32,12 @@ class Version
       class << self
         def current
           @current ||= SCM.tags.last
+
+          if @current.nil? && Util.relative_path('VERSION').exist?
+            @current ||= Version.from_string(Util.relative_path('VERSION').read)
+          end
+
+          @current
```

Comment:
@fmitchell-r7 @athompson-r7 How about the following to provide a clear error message with a *workaround* to this issue?

```ruby
def current
  @current ||= SCM.tags.last
  return @current if @current

  if Util.relative_path('VERSION').exist?
    @current ||= Version.from_string(Util.relative_path('VERSION').read)
  else
    raise 'No current version could be detected. No SCM tags were present and no VERSION file was found.'
  end
end
```

Alternatively in the `else` we default to some other version number.